app/src/main/res/values/strings.xml

```diff
@@ -102,6 +103,7 @@
 	<string name="menu_save">Save</string>
 	<string name="menu_cancel">Cancel</string>
 	<string name="menu_export">Export as GPX</string>
+    <string name="menu_share">Share as GPX</string>
```

Comment:
My studio indented it with 4 spaces, while the other lines are indented with tab. Changed to tab.